src/Controller/AssetsController.php

```diff
@@ -18,12 +20,34 @@
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
 
 class AssetsController extends Controller
 {
     private const WIDTH = 250;
     private const HEIGHT = 170;
 
+    /**
+     * @Route("/assets_files/{id}", requirements={"id": "\d+"}, name="asset_files_url")
```

Comment:
why do you use this controller? it's not really one asset, but user file.

Reply:
I didn't know how to name that thing :s

UserFilesController::getFile(): Response

?

Reply:
What about `MoocController::getFile` ?

Reply:
Is the access to these files supposed to be restricted ?